management-api-server/src/main/java/com/datastax/mgmtapi/resources/NodeOpsResources.java

```diff
@@ -663,6 +663,75 @@ public Response searchIndexRebuild(
         });
   }
 
+  @POST
+  @Path("/encryption/internode/truststore/reload")
+  @Produces(MediaType.TEXT_PLAIN)
+  @ApiResponse(
+      responseCode = "200",
+      description = "Truststore reloaded successfully",
+      content =
+          @Content(
+              mediaType = MediaType.TEXT_PLAIN,
+              schema = @Schema(implementation = String.class),
+              examples = @ExampleObject(value = "OK")))
+  @Operation(summary = "reload truststore", operationId = "reloadTruststore")
+  public Response reloadTruststore() {
+    return handle(
+        () -> {
+          final String releaseVersion =
+              ResponseTools.getSingleRowStringResponse(
+                  app.dbUnixSocketFile, app.cqlService, CASSANDRA_VERSION_CQL_STRING);
+          if (!releaseVersion.startsWith("4.0.0.68") && !releaseVersion.startsWith("4.0.0.69")) {
+            // rebuilding search index is only possible on DSE
+            return Response.status(Response.Status.BAD_REQUEST)
+                .entity("Reloading the truststore manually is only possible on DSE")
+                .build();
+          }
+
+          app.cqlService.executeCql(
+              app.dbUnixSocketFile, "CALL NodeOps.reloadInternodeEncryptionTruststore()");
+
+          return Response.ok("OK").build();
+        });
+  }
+
+  @GET
+  @Path("/encryption/internode/truststore/issuers")
+  @Produces(MediaType.TEXT_PLAIN)
+  @ApiResponse(
+      responseCode = "200",
+      description = "Got list of truststore issuers",
+      content =
+          @Content(
+              mediaType = MediaType.TEXT_PLAIN,
+              schema = @Schema(implementation = String.class),
```

Comment:
issue:
If we really need the entire X509Certificate object represented as JSON to be returned from the API, then a model should be created in `com.datastax.mgmtapi.resources.models`, similar to how we handle things like [SnapshotDetails](com.datastax.mgmtapi.resources.models). Then your `@Produces` line above would specify [APPLICATION_JSON](https://github.com/k8ssandra/management-api-for-apache-cassandra/blob/master/management-api-server/src/main/java/com/datastax/mgmtapi/resources/NodeOpsResources.java#L353) and the `schema` line here would specify that new model class, not `String.class`.

I know that is a lot more work, but that is the goal for many of the API endpoints in here. Existing endpoints get a bit of a pass since there are existing consumers we don't want to break compatibility with. But for new endpoints that return something less trivial, we should create the proper model classes and schemas here for the API.